src/modules/shared/ProductCard/ProductCard.module.scss

```diff
@@ -4,7 +4,6 @@
 
   .productCard {
     display: flex;
-    // width: 287px;

```

Comment:
but we should add max-width to avoid stretch out our card on all avialable colums

Reply:
I've done it and now everything is in center and does not stretch. However when we have 3 columns in some width the space between columns is maybe too much. I don't think it is fixable if we do it this way.
![image](https://github.com/fe-aug23-team4/nice-gadgets/assets/100444192/ad59d54a-3e56-4b95-ae4b-37a87fcdb1b9)